src/schemes/boundary/dummy_particles/dummy_particles.jl

```diff
@@ -378,75 +414,115 @@ function compute_pressure!(boundary_model, ::Union{PressureMirroring, PressureZe
+    (; pressure_offset) = density_calculator
 
     # Loop over all pairs of particles and neighbors within the kernel cutoff.
     foreach_point_neighbor(system, neighbor_system, system_coords, neighbor_coords,
                            neighborhood_search;
                            points=eachparticle(system)) do particle, neighbor,
                                                            pos_diff, distance
-        adami_pressure_inner!(boundary_model, system, neighbor_system,
-                              v_neighbor_system, particle, neighbor, pos_diff,
-                              distance, viscosity, cache, pressure)
+        boundary_pressure_inner!(boundary_model, density_calculator, system,
+                                 neighbor_system, v,
+                                 v_neighbor_system, particle, neighbor, pos_diff,
+                                 distance, viscosity, cache, pressure, pressure_offset)
     end
 end
 
-@inline function adami_pressure_inner!(boundary_model, system, neighbor_system,
-                                       v_neighbor_system, particle, neighbor, pos_diff,
-                                       distance, viscosity, cache, pressure)
+@inline function boundary_pressure_inner!(boundary_model,
+                                          boundary_density_calculator,
+                                          system, neighbor_system::FluidSystem, v,
+                                          v_neighbor_system, particle, neighbor, pos_diff,
+                                          distance, viscosity, cache, pressure,
+                                          pressure_offset)
     density_neighbor = particle_density(v_neighbor_system, neighbor_system, neighbor)
 
-    resulting_acc = neighbor_system.acceleration -
-                    current_acceleration(system, particle)
+    resulting_acceleration = neighbor_system.acceleration -
+                             current_acceleration(system, particle)
 
     kernel_weight = smoothing_kernel(boundary_model, distance)
 
-    pressure[particle] += (particle_pressure(v_neighbor_system, neighbor_system,
-                                             neighbor) +
-                           dot(resulting_acc, density_neighbor * pos_diff)) *
-                          kernel_weight
+    pressure[particle] += (pressure_offset
+                           +
+                           particle_pressure(v_neighbor_system, neighbor_system,
+                                             neighbor)
+                           +
+                           dynamic_pressure(boundary_density_calculator, density_neighbor,
+                                            v, v_neighbor_system, particle, neighbor,
+                                            system, neighbor_system)
+                           +
+                           dot(resulting_acceleration,
+                               density_neighbor * pos_diff)) * kernel_weight
```

Comment:
```suggestion
    # Bernoulli's principle
    stagnation_pressure = pressure_offset +
                          particle_pressure(v_neighbor_system, neighbor_system, neighbor) +
                          dynamic_pressure(boundary_density_calculator, density_neighbor,
                                           v, v_neighbor_system, particle, neighbor,
                                           system, neighbor_system)

    hydraulic_head = dot(resulting_acceleration, density_neighbor * pos_diff)

    pressure[particle] += (stagnation_pressure + hydraulic_head) * kernel_weight
```

Reply:
But this is now used by both the adami and bernoulli pressure condition? Is this not just confusing?